test/solidity/account/nonExisting.js

```diff
@@ -0,0 +1,44 @@
+const { expect, assert } = require('chai')
+const { ethers } = require('hardhat')
+const Constants = require('../../constants')
+
+describe('Solidity Account non Existing', function () {
+  let contract, wallet
+
+  before(async function () {
+    const signers = await ethers.getSigners()
+    wallet = signers[0]
```

Comment:
we should create and use one random address instead of shifting the current wallet address in each test
```typescript
const randomAddress = ethers.Wallet.createRandom().address;
```